fuel/transformers/__init__.py

```diff
@@ -200,31 +325,28 @@ def __init__(self, data_stream, dtype, **kwargs):
         if dtype == 'floatX':
             dtype = config.floatX
         self.dtype = dtype
-        super(Cast, self).__init__(data_stream, **kwargs)
+        kwargs.setdefault('axis_labels', data_stream.axis_labels)
```

Comment:
(Also note here that they can still manually set the `axis_labels` attribute if they really know what they're doing, but I think it's good if the constructor forces something that's sane 99% of the time)